spec/mongo/uri_spec.rb

```diff
@@ -1278,12 +1394,14 @@
     end
 
     context 'when an app name option is provided' do
-      let(:options) { "appname=uri_test" }
+      let(:options) { "appName=uri_test" }
```

Comment:
Or is it canonical casing? Whichever way it is should also be documented.

Reply:
I'm gonna put this back the way it was.